src/component/Calendar/Calendar.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import { ko } from 'date-fns/locale';
+import { CalendarWrapper } from './CalendarWrapper';
+import { NextLink } from './NextLink';
+
+import styled from 'styled-components';
+import 'react-datepicker/dist/react-datepicker.css';
+import { flexSet } from '../../styles/Mixins';
+
+import { useFetch } from '../../hooks';
+import { calcDateRange, dateConversion } from './CalendarLogic';
+
+export const Calendar = ({
+  priceDisplay,
+  excludeOn,
+  setCalendarOff,
+  stayLocation,
+  linkButtonText,
+  linkUrl,
+  redirectComponent,
+  setStayDate,
+}) => {
+  const [dateRange, setDateRange] = useState([null, null]);
+  const [startDate, endDate] = dateRange;
+  const [bookInfo] = useFetch('http://localhost:3000/data/CALENDAR.json', {
+    method: 'GET',
+  });
+
+  const excludeDay = bookInfo => {
+    let arr = [];
+    for (const date in bookInfo.result) {
+      if (bookInfo.result[date].quantity === 0) {
+        arr.push(new Date(date));
+      }
+    }
+    return arr;
+  };
+
+  const renderDayContents = (day, date) => {
+    const tooltipText = `Tooltip for date: ${date}`;
+    return (
+      <>
+        <span title={tooltipText}>{date.getDate()}</span>
+        {bookInfo.result ? (
+          <span title={tooltipText}>
+            {bookInfo.result[dateConversion(date)]
+              ? '₩ ' +
+                bookInfo.result[dateConversion(date)].price.toLocaleString()
+              : null}
+          </span>
+        ) : null}
+      </>
+    );
+  };
+
+  let bookDisable = excludeDay(bookInfo);
+
+  return (
+    <CalendarWrapper>
+      <Button onClick={setCalendarOff}>X</Button>
+      <StyledDatePicker
+        locale={ko}
+        dateFormat="yyyy-mm-dd"
+        selectsRange={true}
```

Comment:
```suggestion
        selectsRange
```
boolean props들은 = true할당안하고 이름만 적어줘도 true로 적용됩니다.